tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -182,7 +182,7 @@ static void SavePixelsToFile(LPCVOID pPixels, DXGI_FORMAT format, UINT32 m_width
 
   VERIFY_SUCCEEDED(ctx.Init());
   VERIFY_SUCCEEDED(CoCreateInstance(CLSID_WICImagingFactory, NULL, CLSCTX_INPROC_SERVER, IID_IWICImagingFactory, (LPVOID*)&pFactory));
-  VERIFY_SUCCEEDED(CoGetMalloc(1, &pMalloc));
+  VERIFY_SUCCEEDED(DxcCoGetMalloc(1, &pMalloc));
```

Comment:
It turns out this is dead code (pMalloc is unused), so you can get rid of it, and get rid of the dependency too!

Reply:
Nice :)